lib/pages/magic_link/join_workspace_magic_link_page.dart

```diff
@@ -186,4 +204,54 @@ class _JoinWorkSpaceMagicLinkPageState
       Logger().e('ERROR during opening console page:\n$e');
     }
   }
+
+  Widget _loadingLayout() {
+    return Container(
+      child: CircularProgressIndicator(
+        backgroundColor: Colors.transparent,
+        color: const Color(0xff004dff),
+        strokeWidth: 1.0,
+      ),
+    );
+  }
+
+  Future<void> _showConfirmLogoutDialog({required String currentServerUrl}) async {
+    if(!mounted) return;
+    await showDialog(
+      context: context,
+      barrierDismissible: false,
+      builder: (BuildContext context) {
+        return WillPopScope(
+          onWillPop: () async => false,
+          child: ConfirmDialog(
+            body: Text(
+              AppLocalizations.of(context)?.confirmLogoutMagicLink(currentServerUrl) ?? '',
```

Comment:
Is it will be ok with other languages that take up much more space than English

Reply:
Have no translated language yet. But it will be ok when switch to other languages (the dialog will be longer of course)